src/main/java/com/sourcegraph/cody/PromptPanel.kt

```diff
@@ -312,8 +312,14 @@ class PromptPanel(project: Project, private val chatSession: ChatSession) : JLay
 data class DisplayedContextFile(val contextItem: ContextItem) {
   override fun toString(): String {
     val contextItemFile = contextItem as? ContextItemFile
+    val isIgnored = contextItemFile?.isIgnored == true
     val isTooLarge = contextItemFile?.title == "large-file" || contextItemFile?.isTooLarge == true
-    val warnIfNeeded = if (isTooLarge) "<i> - ⚠ File too large</i>" else ""
+    val warnIfNeeded =
+        when {
+          isIgnored -> "<i> - ⚠ Ignored by an admin setting.</i>"
+          isTooLarge -> "<i> - ⚠ File too large</i>"
```

Comment:
for consistency - pls either user a dot at the end in the second case or remove it from the first message